src/components/BackOffice/hooks/useSelectContract.js

```diff
@@ -46,7 +46,7 @@ const useSelectContract = () => {
   }, [selectedContractor])
 
   const handleSelectContractor = (e) => {
-    const contractorName = e ? e.target?.value : undefined
+    const contractorName = e.target.value
```

Comment:
This was changed in PR #747 , now being set back to it as handleSelectContractor is no longer called when the reset form happens. Instead we call the new function handleFormReset which has the only task to reset the 2 values (contractor, contract)